ruby-challenges.rb

```diff
@@ -10,6 +10,16 @@
 letter_t = 't'
 # Expected output: ['tea', 'water', 'soda water']
 
+ def array_words_letter(array, letter)
+    array.select do |value|
+     if value.include?(letter)
+             value
+            end 
+        end
+     end
+p array_words_letter(beverages_array, letter_o)
+p array_words_letter(beverages_array, letter_t)
```

Comment:
Looks great!  Since select is already going to filter through the array, you don't need to have an if statement.  value.include?(letter) will return just what you need in the select block.